### PR TITLE
9.9 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/releasenotes/studio-pro/9.9.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ We have also added a setting that lets you configure which styling files should
 * We improved the metadata of [published OData services](/refguide/published-odata-services). Updatable OData resources are no longer annotated with **NonUpdatableNavigationProperties** when that collection is empty.
 * At startup, we now rename the **MxAdmin** user in the database to sync with the username defined in the model.
 * When you open an app from a network path or from a drive in a Mac (both of which will contain double backslashes (`\\`) in the beginning of the file path), a warning is shown to tell you about potential issues that might cause and asks if you wants to proceed anyway.
-* We corrected the language selection list for translation. [**DETAILS? https://github.com/mendix/docs/pull/3943/files**]
+* We updated the region selection list for [supported languages](/refguide/language-settings).
```
Comment (MarkvanMents) [inline content/releasenotes/studio-pro/9.9.md:46]: According to [SPX-2323] this is going to be in 9.10, not 9.9.
Will ask for a review there.

Comment (Adam-Dupaski) [reply content/releasenotes/studio-pro/9.9.md:46]: @MarkvanMents yes please do, also via recent email - original RN was added by @GulamY

Comment (MarkvanMents) [reply content/releasenotes/studio-pro/9.9.md:46]: Removing this line as it has been delayed to 9.10.

Comment (MarkvanMents) [reply content/releasenotes/studio-pro/9.9.md:46]: Removing line.
```suggestion
```

**Pull request merged**